sdk/storage/azure-storage-file-datalake/samples/datalake_samples_access_control_recursive_async.py

```diff
@@ -58,9 +60,19 @@ async def progress_callback(acl_changes):
         failed_entries.append(acl_changes.batch_failures)
 
     # illustrate the operation by using a small batch_size
-    acl_change_result = await directory_client.set_access_control_recursive(acl=acl,
-                                                                            progress_callback=progress_callback,
-                                                                            batch_size=5)
+    try:
+        acl_change_result = await directory_client.set_access_control_recursive(acl=acl,
+                                                                                progress_callback=progress_callback,
+                                                                                batch_size=5)
+    except AzureError:
+        # if the error has continuation_token, you can restart the operation using that continuation_token
+        if AzureError.continuation_token:
```

Comment:
```suggestion
    except AzureError as e:
        # if the error has continuation_token, you can restart the operation using that continuation_token
        if e.continuation_token:
```

Reply:
thanks for catching this I will add a commit to address this!